frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/import-providers-tab/providers/vm-import-password.tsx

```diff
@@ -96,6 +97,12 @@ const stateToProps = (state, { wizardReduxID, provider }) => {
     );
   }
 
+  if (provider === VMImportProvider.VMWARE && hasAllPrerequisiteValuesFiled) {
```

Comment:
we don't have to do the test second time. You can just check if the validation is Error type.And you can do this for both providers with iGetProviderFieldAttribute